utils.ts

```diff
@@ -33,4 +33,6 @@ export function PyNone(): Value {
 export const NUM : Type = {tag: "number"};
 export const BOOL : Type = {tag: "bool"};
 export const NONE : Type = {tag: "none"};
+export const TYPE_VAR : Type = {tag: "type-var"};
 export function CLASS(name : string) : Type {return {tag: "class", name}};
+export function GENERIC_CLASS(name : string, genericArgs: Array<Type>) : Type {return {tag: "class", name, genericArgs}};
```

Comment:
Could we have function GENERIC_CLASS merged into function CLASS with default genericArgs = null? just an idea.